Export error -- "panic: interface conversion: interface {} is nil, not string"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

After running experiment locally, I tried to run `empirica export`. I get the error shown below

### Steps To Reproduce

1. create new test experiment `empirica create test-exp`
2. copy `tajriba.json` (below) to new `.empirica/local/tajriba.json`
3. run `empirica export` in `test-exp` directory. 
4. get error. 

### Empirica Version

```text
Version: v1.11.3
SHA:     c9e8647
Build:   218
Branch:  main
Time:    2024-07-15T01:58:21Z

Client:  1.11.3
Server:  1.11.3
```


### What OS are you seeing the problem on?

Windows

### What browser are you seeing the problem on?

Does not apply

### Relevant log output

```shell
15:27:23.269 INF Starting CSV export... output=/home/jpwhite/GitHub/others/test-exp/test-exp-2024-07-15-15-27-23.zip tajriba=/home/jpwhite/GitHub/others/test-exp/.empirica/local/tajriba.json
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/empiricaly/empirica/internal/export.prepare({0xc00012e820?, 0xc000170800?})
        /build/internal/export/export.go:94 +0x1a25
github.com/empiricaly/empirica/internal/export.ExportCSV({0xc00012e820?, 0xd85d5e?}, {0xc00012e910, 0x45})
        /build/internal/export/csv.go:16 +0x52
github.com/empiricaly/empirica/cmds/empirica/cmd.addExportCommand.func1(0xc0001a3180, {0x1515bc0, 0x0, 0xd72271?})
        /build/cmds/empirica/cmd/export.go:89 +0x447
github.com/spf13/cobra.(*Command).execute(0xc0001a3180, {0x1515bc0, 0x0, 0x0})
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x6aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc00051d400)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x38d
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/empiricaly/empirica/cmds/empirica/cmd.Execute()
        /build/cmds/empirica/cmd/root.go:144 +0x236
main.main()
        /build/cmds/empirica/main.go:10 +0xf
```


### Anything else?

Here is the tajriba file that gets the error: [tajriba.json](https://github.com/user-attachments/files/16239494/tajriba.json). 
I have tried it different times with different tajriba files and it doesn't work for any. Interesingly, `empirica export` did work previously on my project. But as far as I can tell nothing has changed :/


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.changeset/export-null-attribute.md
+++ b/.changeset/export-null-attribute.md
@@ -1,0 +1,8 @@
+---
+"@empirica/core": patch
+---
+
+Support export of `null` attributes.
+
+Previously, `null` attributes would crash the export process. Now, they are
+exported as `null` in the CSV file.

--- a/go.mod
+++ b/go.mod
@@ -47,6 +47,7 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
+	github.com/go-bindata/go-bindata v3.1.2+incompatible // indirect
 	github.com/go-logr/logr v1.2.4 // indirect
 	github.com/go-playground/locales v0.14.0 // indirect
 	github.com/go-playground/universal-translator v0.18.0 // indirect

--- a/internal/export/export.go
+++ b/internal/export/export.go
@@ -88,11 +88,28 @@ func prepare(tajfile string) ([]*Kind, error) {
 
 			scopeKinds[obj["id"].(string)] = kinds[obj["kind"].(string)]
 		case "Attribute":
-			obj := data["obj"].(map[string]interface{})
+			obj, ok := data["obj"].(map[string]interface{})
+			if !ok {
+				continue
+			}
 
-			key := obj["key"].(string)
-			val := cast(obj["val"].(string))
-			createdAt := obj["createdAt"].(string)
+			key, ok := obj["key"].(string)
+			if !ok {
+				continue
+			}
+
+			valRaw, ok := obj["val"].(string)
+			if !ok {
+				valRaw = "null"
+			}
+
+			val := cast(valRaw)
+
+			createdAt, ok := obj["createdAt"].(string)
+			if !ok {
+				continue
+			}
+
 			vector, _ := obj["vector"].(bool)
 			indexf, hasIndex := obj["index"].(float64)
 			index := int(indexf)
@@ -112,7 +129,10 @@ func prepare(tajfile string) ([]*Kind, error) {
 				continue
 			}
 
-			nodeID := obj["nodeID"].(string)
+			nodeID, ok := obj["nodeID"].(string)
+			if !ok {
+				return nil, errors.New("nodeID not found")
+			}
 
 			kind, ok := scopeKinds[nodeID]
 			if !ok {

